### DISCUSSION TITLE
FOSS master-server?

### DISCUSSION THREAD

**Discussion by poVoq** [piqueserver/piqueserver#720] [Q&A]
Is there any FOSS implementation of a master-server or are there any plans to make one?

Would be nice to have one with servers only for certain game-modes and OpenSpades support.

**Comment by bananasov** [comment]
If there was a FOSS master server it would divide the community even more than it already it.

**Comment by poVoq** [comment]
Maybe, but if one would want to develop OpenSpades further beyond AoS backwards compatibility then it would be bad to continue using the same master-server.

**Comment by bananasov** [comment]
There is a low chance that any one would do that.

**Comment by NotAFile** [comment]
I wrote half of one, I might upload it at some point. The protocol is pretty simple

**Reply by bananasov** [reply]
In which language is it made in if you don't mind me asking?

**Reply by NotAFile** [reply]
Python, but it's only like 100 lines of pretty simple code so it doesn't really matter

**Reply by poVoq** [reply]
Yes please share it, would be cool that have that just in case.
